packages/block-library/src/social-links/block.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "core/social-links",
+	"category": "widgets",
+	"icon": "share",
```

Comment:
I see that `icon` is also declared in `social-links/index.js`. Do we need it in both places?

Reply:
Good catch, removed the duplicate here.